src/components/base/widget-position.js

```diff
@@ -332,6 +333,14 @@ export default WrappedComponent =>
 					opacity: 1,
 				});
 			});
+
+			if(domElement.$){
```

Comment:
nit: space after the `if `

And also after the `=>` below.

You should probably run `npm run format`.

Reply:
Hello @wincent,

Thank you, I made the format changes. I was unable to run npm run format though.